.github/workflows/conformance.yml

```diff
@@ -108,5 +111,5 @@ jobs:
 
         echo "Running tests for Test${KIND_UPPER}Conformance/${KIND}/${NAME} ... "
 
-        go test -v -tags=conftests -count=1 ./tests/... \
+        go test -v -tags=conftests -count=1 ./tests/conformance \
```

Comment:
this will only run the conformance tests and not the unit tests within `tests` directory.